### PR TITLE
TS chat

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/chat/conversation/messages/coinflip/index.tsx [hunk 1]
```diff
@@ -83,9 +82,6 @@ class CoinFlip extends React.Component<Props, State> {
             <Kb.Text selectable={true} type="BodySmallSemibold">
               {!Styles.isMobile && 'Collecting '}secrets: {this._revealSummary()}
             </Kb.Text>
-            {this.props.phase === 'complete' && (
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/coinflip/index.tsx]: @adamjspooner ts catches that this can never be true. this was added about 3 months ago

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/giphy/container.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  conversationIDKey: Types.ConversationIDKey
+}
+
+const mapStateToProps = (state, ownProps: OwnProps) => {
+  const {conversationIDKey} = ownProps
+  const giphy = state.chat2.giphyResultMap.get(conversationIDKey, null)
+  return {
+    // Auto generated from flowToTs. Please clean me!
+    galleryURL:
+      // Auto generated from flowToTs. Please clean me!
```
Comment (mmaxim) [inline shared/chat/conversation/giphy/container.tsx]: This got pretty butchered.

File: shared/chat/conversation/maybe-mention/team-container.tsx [hunk 3]
```diff
@@ -25,15 +24,36 @@ const mapStateToProps = (state, {allowFontScaling, name, channel, style}) => {
       ? maybeMentionInfo.team
       : null
   return {
-    _convID: mentionInfo?.convID,
+    // Auto generated from flowToTs. Please clean me!
```
Comment (mmaxim) [inline shared/chat/conversation/maybe-mention/team-container.tsx]: This too

**Pull request merged**